WordPress/Classes/ViewRelated/Post/WPStyleGuide+Posts.swift

```diff
@@ -0,0 +1,140 @@
+        label.attributedText = NSAttributedString(string: title, attributes: WPStyleGuide.postCardTitleAttributes)
+        label.lineBreakMode = .byTruncatingTail
+    }
+
+    class func applyPostSnippetStyle(_ title: String, into label: UILabel) {
+        label.attributedText = NSAttributedString(string: title, attributes: WPStyleGuide.postCardSnippetAttributes)
+        label.lineBreakMode = .byTruncatingTail
+    }
+
+    // MARK: - Card View Styles
+    class func postCardBorderColor() -> UIColor {
+        return UIColor(fromRGBAColorWithRed: 215.0, green: 227.0, blue: 235.0, alpha: 1.0)
+    }
+
+    class func applyPostCardStyle(_ cell: UITableViewCell) {
+        cell.backgroundColor = greyLighten30()
+        cell.contentView.backgroundColor = greyLighten30()
+    }
+
+    class func applyPostTitleStyle(_ label: UILabel) {
+        label.textColor = darkGrey()
+    }
+
+    class func applyPostSnippetStyle(_ label: UILabel) {
+        label.textColor = darkGrey()
+    }
+
+    class func applyPostDateStyle(_ label: UILabel) {
+        configureLabelForRegularFontStyle(label)
+        label.textColor = grey()
+    }
+
+    class func applyPostButtonStyle(_ button: UIButton) {
+        configureLabelForRegularFontStyle(button.titleLabel)
+        button.setTitleColor(grey(), for: .normal)
+    }
+
+    class func applyPostProgressViewStyle(_ progressView: UIProgressView) {
+        progressView.trackTintColor = greyLighten20()
+        progressView.progressTintColor = mediumBlue()
+        progressView.tintColor = mediumBlue()
+    }
+
+    class func applyRestorePostLabelStyle(_ label: UILabel) {
+        configureLabelForRegularFontStyle(label)
+        label.textColor = grey()
+    }
+
+    class func applyRestorePostButtonStyle(_ button: UIButton) {
+        configureLabelForRegularFontStyle(button.titleLabel)
+        button.setTitleColor(wordPressBlue(), for: .normal)
+        button.setTitleColor(darkBlue(), for: .highlighted)
+    }
+
+    class func applyBorderStyle(_ view: UIView) {
+        view.heightAnchor.constraint(equalToConstant: 1.0 / UIScreen.main.scale).isActive = true
+        view.backgroundColor = postCardBorderColor()
+    }
+
+    class func applyActionBarButtonStyle(_ button: UIButton) {
+        button.flipInsetsForRightToLeftLayoutDirection()
+        button.setImage(button.imageView?.image?.imageWithTintColor(grey()), for: .normal)
+        button.setTitleColor(grey(), for: .normal)
+        button.setTitleColor(darkGrey(), for: .highlighted)
+        button.setTitleColor(darkGrey(), for: .selected)
+    }
+
+    class func applySelectedBackgroundView(_ selectedBackgroundView: UIView, topMargin: CGFloat) {
+        let marginMask = UIView()
+        selectedBackgroundView.addSubview(marginMask)
+        marginMask.translatesAutoresizingMaskIntoConstraints = false
+        marginMask.leadingAnchor.constraint(equalTo: selectedBackgroundView.leadingAnchor).isActive = true
```

Comment:
Could we cleanup these into a single `activateConstraints` call?

Reply:
Done!